notebooks/0103_GFED4-BB4CMIP-prepare.py

```diff
@@ -0,0 +1,249 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.4
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+from collections import defaultdict
+
+import cf_xarray.units
+import matplotlib.pyplot as plt
+import numpy as np
+import pint_xarray
+import scmdata
+import xarray as xr
+
+from emissions_harmonization_historical.constants import DATA_ROOT
+
+# %%
+pint_xarray.accessors.default_registry = pint_xarray.setup_registry(cf_xarray.units.units)
+
+# %%
+raw_data_path = DATA_ROOT / "national/gfed-bb4cmip/data_raw/data"
+raw_data_path
+
+# %%
+rcmip_data_path = DATA_ROOT / "global/rcmip/data_raw/rcmip-emissions-annual-means-v5-1-0.csv"
+
+# %% [markdown]
+# Group raw data into variable groups which can be used for processing.
+
+# %%
+bb4cmip_files = list(raw_data_path.rglob("*.nc"))
+bb4cmip_file_groups = defaultdict(list)
+
+for file in bb4cmip_files:
+    variable = file.name.split("_")[0]
+    bb4cmip_file_groups[variable].append(file)
+
+bb4cmip_file_groups.keys()
+
+# %% [markdown]
+# An example of loading data.
+
+# %%
+cell_area = xr.open_mfdataset(bb4cmip_file_groups["gridcellarea"])
+cell_area
+
+# %%
+gas = "BC"
+unit_target = "Mt / yr"
+unit_label = "Mt BC / yr"  # Have to be careful here, depends on the species
+rcmip_gas = "BC"
+
+# gas = "CO"
+# unit_target = "Mt / yr"
+# unit_label = "Mt CO / yr"
+# rcmip_gas = "CO"
+
+# # TODO: check. I don't think we used this last time so we may want to ignore this again.
```

Comment:
> Is it because it is assumed to be part of the global GCB value that we'll harmonize to?

I believe so (we should ask Thomas whether he thinks it's already part of the GCB value or not).

Reply:
OK, turned into issue #17